Components/Badge/badge.html

```diff
@@ -0,0 +1,142 @@
+    <link rel="icon" href="../../Assets/light.png">
+    <link rel="stylesheet" href="../../utilsCSS/getting-started.css" />
+    <link rel="stylesheet" href="./Badge.css" />
+    <script src="https://kit.fontawesome.com/e06444868d.js" crossorigin="anonymous"></script>
+    <title>Badge | Avid</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="logo">
+            <a href="/index.html">
+                <img class="logo" src="../../Assets/light.png" />
+            </a>
+        </div>
+
+        <nav class="navbar">
+            <ul>
+                <a href="/index.html">
+                    <li class="highlight">Home</li>
+                </a>
+                <a href="./getting-started.html">
+                    <li class="highlight">Documentation</li>
+                </a>
+            </ul>
+        </nav>
+    </header>
+    <section class="documentation">
+        <aside class="sidebar">
+            <div class="list-topic">
+                <h4 class="highlight">Get Started</h4>
+            </div>
+            <ul>
+                <li><a href="../../getting-started.html">Introduction</a></li>
+                <!-- <li><a href="../Colors/colors.html">Colors</a></li>
+          <li><a href="../Typography/typography.html">Typography</a></li> -->
+            </ul>
+
+            <div class="list-topic">
+                <h4 class="highlight">Components</h4>
+            </div>
+            <ul>
+                <li><a href="../Cards/card.html">Card</a></li>
+                <li><a href="../Avatar/avatar.html">Avatar</a></li>
+                <!-- <li><a href="./Components/">Alert</a></li> -->
+                <li><a href="">Badge</a></li>
+                <li><a href="../Buttons/buttons.html">Button</a></li>
+                <!-- <li><a href="./Components/>Image</a></li>
+          <li><a href="./Components/">Input</a></li>
+          <li><a href="./Components/>Navigation</a></li>
+          <li><a href="./Components/">Snackbar</a></li> -->
+            </ul>
+        </aside>
+        <main class="main-content">
+            <div class="content-head">
+                <h2 class="highlight"> Badges</h2>
+            </div>
+            <div class="content">
+                <p>
+                    Badges are icons on avatars and illustrations that show there status
+                </p>
+            </div>
+
+
+            <div class="divider"></div>
+            <div class="content-warp">
+                <div class="content-head">
+                    <h2 class="highlight">Profile Avatar</h2>
+                </div>
+                <div class="content">
+                    <div class="example">
+
+                        <div class="badge-container">
+                            <img src="https://pbs.twimg.com/profile_images/1464429641284784135/aD0otj1B_400x400.jpg"
+                                alt="profile-pic" class="avatar lg" />
+                            <div class="badge bg-success"></div>
```

Comment:
I am sure these divs are empty for a reason

Reply:
Yeah, The passed Text will appear inside the badge